single-line-diagram-core/src/main/resources/tautologies.css

```diff
@@ -10,3 +9,20 @@ svg {fill: none; stroke: black}
 .sld-bottom-feeder .sld-label {dominant-baseline: hanging}
 .sld-arrow-p .sld-label {dominant-baseline: middle}
 .sld-arrow-q .sld-label {dominant-baseline: middle}
+/* Fill: none */
+.sld-disconnector {fill: none}
+.sld-busbar-section {fill: none}
+.sld-bus-connection {fill: none}
+.sld-wire {fill: none}
+.sld-load-break-switch {fill: none}
+.sld-load {fill: none}
+.sld-generator {fill: none}
+.sld-two-wt {fill: none}
+.sld-three-wt {fill: none}
+.sld-capacitor {fill: none}
+.sld-inductor {fill: none}
+.sld-pst {fill: none}
+.sld-svc {fill: none}
+.sld-vsc {fill: none}
+/* Stroke blue & fill: white */
+.sld-breaker {stroke: blue; fill: white}
```

Comment:
I think these lines should be in components.css instead. What I meant with this "tautologies" file is that it contains properties that remain true whatever comes next. Which is not the case of the components as they may be filled with a color for a specific component library.